src/main/java/org/sonatype/gradle/plugins/scan/common/PluginVersionUtils.java

```diff
@@ -24,11 +24,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-class PluginVersionUtils
+public class PluginVersionUtils
 {
   private static final Logger log = LoggerFactory.getLogger(PluginVersionUtils.class);
 
-  private static final String PROPERTIES_PATH = "com/sonatype/insight/client.properties";
+  private static final String PROPERTIES_PATH = "nexus/shadow/com/sonatype/insight/client.properties";
```

Comment:
When relocating packages using the `shadow` plugin we're already excluding the `com.sonatype` package, but due to a long standing bug Strings referencing packages are being relocated disregarding the exclusion:

https://github.com/johnrengelman/shadow/issues/305
https://github.com/johnrengelman/shadow/issues/232